packages/components/stories/icon/icons.js

```diff
@@ -32,6 +32,7 @@ export const icons =
         'IcAppstoreSuccess',
         'IcAppstoreTickWhite',
         'IcAppstoreTick',
+        'IcAppstoreTradersHubHomeUpdated',
```

Comment:
These changes are not needed.

Reply:
Sure, I have removed this file. 

Thanks @matin-deriv for raising this 🙇